communication/marketing-team/storytelling-resources/blog-guidelines.md

```diff
@@ -62,4 +62,4 @@ The following are helpful resources for authoring articles:
 
 ## Review Process
 
-This process is bootstrapped from the sig-docs blog [subproject](/sig-docs/blog-subproject/README.md). Please ask for review from community liaisons on the sig-docs-blog team or let the team in [#sig-contribex](https://kubernetes.slack.com/archives/C1TU9EB9S) know you need help.
+This process is initiated in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) and uses the processes from the sig-docs blog [subproject](/sig-docs/blog-subproject/README.md). Please ask for review from community liaisons on the sig-docs-blog team or let the team in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) know you need help.
```

Comment:
```suggestion
This process is initiated in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) and uses the processes from the SIG Docs blog [subproject](/sig-docs/blog-subproject/README.md). Please ask for review from community liaisons on the sig-docs-blog team or let the team in [#sig-contribex-comms](https://kubernetes.slack.com/archives/C03KT3SUJ20) know you need help.
```

Reply:
Good idea, I kept it minimal but we should go there. Please check my latest commit, I've tried to address that without being excessively wordy.